test/nuntiux_test.exs

```diff
@@ -1,23 +1,37 @@
 defmodule NuntiuxTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: false
```

Comment:
This is the default, but Credo complains otherwise.
If we set it to `true` though, we get issues with registering/unregistering those processes in parallel...